dataactbroker/handlers/accountHandler.py

```diff
@@ -135,7 +136,6 @@ def login(self,session):
         except Exception as e:
             # Return 500
             return JsonResponse.error(e,StatusCode.INTERNAL_ERROR)
-        return self.response
```

Comment:
PyCharm said the code couldn't be reached, which is why I removed it. I did double-check and it looked like it wasn't an issue.